data/mods/Magiclysm/items/comestibles.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "type": "COMESTIBLE",
+    "id": "owlbear_egg_stone",
+    "name": "owlbear egg",
+    "weight": 2395,
+    "color": "gray",
+    "spoils_in": "14 days",
+    "comestible_type": "FOOD",
+    "symbol": "0",
+    "quench": 128,
+    "healthy": 1,
+    "calories": 2560,
+    "description": "This egg laid by an owlbear seems almost indistinguishable from a large rock until you touch it and can feel the warm glow of potential life energy from within.  It's hard to believe that an owlbear could grow so quickly from something so small.  Perhaps you could carefully crack it open to get at the liquid golden goodness inside.",
+    "price": 300000,
+    "material": "egg",
+    "volume": 10,
+    "stack_size": 1,
+    "fun": -18,
+    "flags": [ "FREEZERBURN", "INEDIBLE", "CATTLE" ],
+    "//": "This is kludgy. A post-thresh cattle mutant who doesn't have herbivore could eat this egg but that's a pretty narrow case. The only more complete way I know to make it inedible would be to give it the powder material but then that would be more obvious to players.",
+    "rot_spawn": "GROUP_EGG_OWLBEAR",
+    "vitamins": [ [ "vitA", 290 ], [ "calcium", 97 ], [ "iron", 129 ], [ "vitB", 677 ] ],
+    "rot_spawn_chance": 80,
+    "//": "Alchemically speaking, I see this egg as having the elements of Growth, Stone, and Egg. Growth would make things grow faster, the simplest example being a magical liquid fertilizer. Stone would be a defensive element, used in creating something like a potion of stoneskin. Egg would be interesting though. What if instead of being imbibed there were potions that instead functioned similarly to the sourdough items, or the blob vehicle parts, in that they take a gestation period and then you could activate them and have a new item pop out. Creatures sure but especially biomancers could have temporarily-lived tools, weapons, and armor."
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_owlbear",
+    "name": "owlbear egg",
+    "copy-from": "egg_chicken",
+    "rot_spawn": "GROUP_EGG_OWLBEAR",
+    "phase": "liquid",
+    "healthy": 3,
```

Comment:
382 health from one owlbear egg? (128 * 3)
is that reasonable?

Reply:
No, but the eggs aren't entirely reasonable at the moment anyways. 128 * 1 is still pretty absurd. For flavor purposes I want them to have higher health than chicken eggs but the main problem is that I got really stuck on having the liquid contents be otherwise the same as chicken eggs , and chicken eggs are way too small right now. I think a better solution would be to put owlbear eggs at a reasonable volume and separately raise an issue on bird egg sizes. Or would it be better to start from a PR with that since it's something I know how to change myself?

Reply:
i don't really like the hacky approach to "uncrafting an egg into 128 liquid portions" very much. i am starting to think we should just add some "owlbear egg" recipes for scrambled eggs and boiled eggs, because i'd rather have owlbear eggs be primarily an alchemical component.

Reply:
That also works. I thought this could have been an elegant way to avoid having to add extra recipe clutter but it's possible I was being too clever for my own good. Though for what it's worth it'd be more like 40 portions when I fix the numbers, and I think I have the egg set up to primarily be an alchemical component already.

Reply:
One last thought for the night: these are (/will be) very rare and inherently magical items. Especially once they have alternative, competing uses I don't think it's unreasonable for them to be a rather exceptional prize.